src/libraries/Common/src/System/Net/Security/NetEventSource.Security.Windows.cs

```diff
@@ -9,87 +9,178 @@ namespace System.Net
 {
     internal sealed partial class NetEventSource
     {
+        private const int EnumerateSecurityPackagesId = NextAvailableEventId + 1;
+        private const int SspiPackageNotFoundId = EnumerateSecurityPackagesId + 1;
+        private const int AcquireDefaultCredentialId = SspiPackageNotFoundId + 1;
+        private const int AcquireCredentialsHandleId = AcquireDefaultCredentialId + 1;
+        private const int InitializeSecurityContextId = AcquireCredentialsHandleId + 1;
+        private const int SecurityContextInputBufferId = InitializeSecurityContextId + 1;
+        private const int SecurityContextInputBuffersId = SecurityContextInputBufferId + 1;
+        private const int AcceptSecuritContextId = SecurityContextInputBuffersId + 1;
+        private const int OperationReturnedSomethingId = AcceptSecuritContextId + 1;
+        // Make sure to update the event IDs in NetEventSource.Security.cs if you add more events here
```

Comment:
I see NetEventSource.Security.cs got deleted... or is it about the one in System.Net.Security?

Reply:
I removed the `NetEventSource.Security.cs` from the Common folder and moved its contents into `NetEventSource.Security.Windows.cs`.

Yes, the comment is referring to the `System.Net.Security` version (which is now the only one).